gui/src/objects/food/Food.cpp

```diff
@@ -1,6 +1,4 @@
 /*
-** EPITECH PROJECT, 2024
```

Comment:
Sorry I just meant, removing 'WSL ...' the Epitech tag and project name shall stay